app/src/main/res/layout/logged_data_item.xml

```diff
@@ -73,6 +74,18 @@
             app:layout_constraintEnd_toStartOf="@+id/focus_play"
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/ic_card_mem_graph" />
+
+        <ImageView
+            android:id="@+id/delete_file_btn"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="8dp"
```

Comment:
Remove hard coding.

Reply:
@jddeep Done with the changes! Please check.